### ISSUE TITLE
Getting 403 on garden deploy for ingress-controller while following getting started tutorial

### ISSUE THREAD

**Issue by samirspatel** [garden-io/garden#2177]
### Current Behavior
`garden deploy` causes a 403

### Expected behavior
It works

### Reproducible example
Follow the intro and get to step 4 https://docs.garden.io/getting-started/4-deploy-and-test

```
garden deploy
Deploy 🚀

 providers                 → Preparing environment...
    local-kubernetes          → Configuring...
   ✔ providers                 → Getting status... → Cached
      ℹ Run with --force-refresh to force a refresh of provider statuses.
   ✔ providers                 → Getting status... → Cached
      ℹ Run with --force-refresh to force a refresh of provider statuses.
   ✔ default-backend           → Getting build status for v-63466f0842... → Done (took 0 sec)
   ✖ ingress-controller        → Building version v-ec25112937...
   ✔ default-backend           → Deploying version v-63466f0842... → Already deployed

Failed resolving provider local-kubernetes. Here is the output:
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
local-kubernetes — an error occurred when configuring environment:
Error: Command "/Users/samirpatel/.garden/tools/helm/4d13a6c3bb98b810/darwin-amd64/helm --kube-context docker-desktop pull stable/nginx-ingress --untar --untardir
/var/folders/r0/vqtrb1_x3ylc0vs7l38ndlrh0000gn/T/tmp-47363-oD0EbCZvUKQ0 --version 1.26.2" failed with code 1:

Error: failed to fetch https://kubernetes-charts.storage.googleapis.com/nginx-ingress-1.26.2.tgz : 403 Forbidden

Error: failed to fetch https://kubernetes-charts.storage.googleapis.com/nginx-ingress-1.26.2.tgz : 403 Forbidden
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Failed resolving one or more providers:
- local-kubernetes

See error.log for detailed error message
```

### Additional context

**project.garden.yml**
```
kind: Project
name: demo-project
environments:
  - name: default
providers:
  - name: local-kubernetes
```

**frontend/garden.yml**
```
kind: Module
type: container
name: frontend
dockerfile: Dockerfile
services:
  - name: frontend
    ports:
      - name: http
        containerPort: 8080
    ingresses:
      - path: /hello-frontend
        port: http
      - path: /call-backend
        port: http
    dependencies:
      - backend
tests:
  - name: unit
    args: [npm, test]
  - name: integ
    args: [npm, run, integ]
    dependencies:
      - frontend

```

**backend/garden.yml**
```
kind: Module
type: container
name: backend
dockerfile: Dockerfile
services:
  - name: backend
    ports:
      - name: http
        containerPort: 8080
        servicePort: 80
    ingresses:
      - path: /hello-backend
        port: http
```

### Your environment
Docker desktop 3.0.3 Local Kubernetes Cluster

* **OS:** 
* **How I'm running Kubernetes:**  

`garden version` `0.12.11`

**Comment by samirspatel** [comment]
error.log
```

[2021-01-03T21:55:59.748Z] local-kubernetes — an error occurred when configuring environment:
Error: Command "/Users/samirpatel/.garden/tools/helm/4d13a6c3bb98b810/darwin-amd64/helm --kube-context docker-desktop pull stable/nginx-ingress --untar --untardir /var/folders/r0/vqtrb1_x3ylc0vs7l38ndlrh0000gn/T/tmp-57610-A8KsoCTxS45t --version 1.26.2" failed with code 1:

Error: failed to fetch https://kubernetes-charts.storage.googleapis.com/nginx-ingress-1.26.2.tgz : 403 Forbidden

Error: failed to fetch https://kubernetes-charts.storage.googleapis.com/nginx-ingress-1.26.2.tgz : 403 Forbidden

Error Details:


results:
  stage-build.default-backend:
    type: stage-build
    key: stage-build.default-backend
    name: default-backend
    description: staging build for default-backend
    completedAt: 2021-01-03T21:55:54.214Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output: {}
    dependencyResults: {}
    version: v-63466f0842
    startedAt: 2021-01-03T21:55:54.105Z
  build.default-backend:
    type: build
    key: build.default-backend
    name: default-backend
    description: building default-backend
    completedAt: 2021-01-03T21:55:54.230Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output:
      fresh: false
    dependencyResults:
      stage-build.default-backend:
        type: stage-build
        key: stage-build.default-backend
        name: default-backend
        description: staging build for default-backend
        completedAt: 2021-01-03T21:55:54.214Z
        batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
        output: {}
        dependencyResults: {}
        version: v-63466f0842
        startedAt: 2021-01-03T21:55:54.105Z
    version: v-63466f0842
    startedAt: 2021-01-03T21:55:54.215Z
  stage-build.ingress-controller:
    type: stage-build
    key: stage-build.ingress-controller
    name: ingress-controller
    description: staging build for ingress-controller
    completedAt: 2021-01-03T21:55:54.230Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output: {}
    dependencyResults: {}
    version: v-ec25112937
    startedAt: 2021-01-03T21:55:54.105Z
  get-service-status.default-backend:
    type: get-service-status
    key: get-service-status.default-backend
    name: default-backend
    description: >-
      getting status for service 'default-backend' (from module
      'default-backend')
    completedAt: 2021-01-03T21:55:54.307Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output:
      forwardablePorts: []
      ingresses: []
      state: ready
      version: v-63466f0842
      detail:
        remoteResources: []
        workload:
          kind: Deployment
          apiVersion: apps/v1
          metadata:
            name: default-backend
            annotations:
              garden.io/configured.replicas: '1'
              garden.io/generated: 'true'
              garden.io/version: v-63466f0842
            namespace: garden-system
            labels:
              module: default-backend
              service: default-backend
          spec:
            selector:
              matchLabels:
                service: default-backend
            template:
              metadata:
                labels:
                  module: default-backend
                  service: default-backend
              spec:
                containers: []
                restartPolicy: Always
                terminationGracePeriodSeconds: 5
                dnsPolicy: ClusterFirst
            replicas: 1
            strategy:
              type: RollingUpdate
              rollingUpdate:
                maxUnavailable: 1
                maxSurge: 1
            revisionHistoryLimit: 3
      outputs: {}
    dependencyResults: {}
    version: v-63466f0842
    startedAt: 2021-01-03T21:55:54.101Z
  deploy.default-backend:
    type: deploy
    key: deploy.default-backend
    name: default-backend
    description: deploying service 'default-backend' (from module 'default-backend')
    completedAt: 2021-01-03T21:55:54.317Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output:
      forwardablePorts: []
      ingresses: []
      state: ready
      version: v-63466f0842
      detail:
        remoteResources: []
        workload:
          kind: Deployment
          apiVersion: apps/v1
          metadata:
            name: default-backend
            annotations:
              garden.io/configured.replicas: '1'
              garden.io/generated: 'true'
              garden.io/version: v-63466f0842
            namespace: garden-system
            labels:
              module: default-backend
              service: default-backend
          spec:
            selector:
              matchLabels:
                service: default-backend
            template:
              metadata:
                labels:
                  module: default-backend
                  service: default-backend
              spec:
                containers: []
                restartPolicy: Always
                terminationGracePeriodSeconds: 5
                dnsPolicy: ClusterFirst
            replicas: 1
            strategy:
              type: RollingUpdate
              rollingUpdate:
                maxUnavailable: 1
                maxSurge: 1
            revisionHistoryLimit: 3
      outputs: {}
    dependencyResults:
      get-service-status.default-backend:
        type: get-service-status
        key: get-service-status.default-backend
        name: default-backend
        description: >-
          getting status for service 'default-backend' (from module
          'default-backend')
        completedAt: 2021-01-03T21:55:54.307Z
        batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
        output:
          forwardablePorts: []
          ingresses: []
          state: ready
          version: v-63466f0842
          detail:
            remoteResources: []
            workload:
              kind: Deployment
              apiVersion: apps/v1
              metadata:
                name: default-backend
                annotations:
                  garden.io/configured.replicas: '1'
                  garden.io/generated: 'true'
                  garden.io/version: v-63466f0842
                namespace: garden-system
                labels:
                  module: default-backend
                  service: default-backend
              spec:
                selector:
                  matchLabels:
                    service: default-backend
                template:
                  metadata:
                    labels:
                      module: default-backend
                      service: default-backend
                  spec:
                    containers: []
                    restartPolicy: Always
                    terminationGracePeriodSeconds: 5
                    dnsPolicy: ClusterFirst
                replicas: 1
                strategy:
                  type: RollingUpdate
                  rollingUpdate:
                    maxUnavailable: 1
                    maxSurge: 1
                revisionHistoryLimit: 3
          outputs: {}
        dependencyResults: {}
        version: v-63466f0842
        startedAt: 2021-01-03T21:55:54.101Z
      build.default-backend:
        type: build
        key: build.default-backend
        name: default-backend
        description: building default-backend
        completedAt: 2021-01-03T21:55:54.230Z
        batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
        output:
          fresh: false
        dependencyResults:
          stage-build.default-backend:
            type: stage-build
            key: stage-build.default-backend
            name: default-backend
            description: staging build for default-backend
            completedAt: 2021-01-03T21:55:54.214Z
            batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
            output: {}
            dependencyResults: {}
            version: v-63466f0842
            startedAt: 2021-01-03T21:55:54.105Z
        version: v-63466f0842
        startedAt: 2021-01-03T21:55:54.215Z
    version: v-63466f0842
    startedAt: 2021-01-03T21:55:54.312Z
  get-service-status.ingress-controller:
    type: get-service-status
    key: get-service-status.ingress-controller
    name: ingress-controller
    description: >-
      getting status for service 'ingress-controller' (from module
      'ingress-controller')
    completedAt: 2021-01-03T21:55:54.922Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    output:
      forwardablePorts: []
      state: missing
      detail: {}
      outputs: {}
    dependencyResults:
      get-service-status.default-backend:
        type: get-service-status
        key: get-service-status.default-backend
        name: default-backend
        description: >-
          getting status for service 'default-backend' (from module
          'default-backend')
        completedAt: 2021-01-03T21:55:54.307Z
        batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
        output:
          forwardablePorts: []
          ingresses: []
          state: ready
          version: v-63466f0842
          detail:
            remoteResources: []
            workload:
              kind: Deployment
              apiVersion: apps/v1
              metadata:
                name: default-backend
                annotations:
                  garden.io/configured.replicas: '1'
                  garden.io/generated: 'true'
                  garden.io/version: v-63466f0842
                namespace: garden-system
                labels:
                  module: default-backend
                  service: default-backend
              spec:
                selector:
                  matchLabels:
                    service: default-backend
                template:
                  metadata:
                    labels:
                      module: default-backend
                      service: default-backend
                  spec:
                    containers: []
                    restartPolicy: Always
                    terminationGracePeriodSeconds: 5
                    dnsPolicy: ClusterFirst
                replicas: 1
                strategy:
                  type: RollingUpdate
                  rollingUpdate:
                    maxUnavailable: 1
                    maxSurge: 1
                revisionHistoryLimit: 3
          outputs: {}
        dependencyResults: {}
        version: v-63466f0842
        startedAt: 2021-01-03T21:55:54.101Z
    version: v-ec25112937
    startedAt: 2021-01-03T21:55:54.308Z
  build.ingress-controller:
    type: build
    description: building ingress-controller
    key: build.ingress-controller
    name: ingress-controller
    startedAt: 2021-01-03T21:55:54.230Z
    completedAt: 2021-01-03T21:55:59.651Z
    batchId: 0b07fd62-264e-4a38-a365-b9ad59b9f9c6
    version: v-ec25112937


[2021-01-03T21:55:59.795Z] Failed resolving one or more providers:
- local-kubernetes

Error Details:

rawConfigs: []
taskResults:
  resolve-provider.exec:
    type: resolve-provider
    key: resolve-provider.exec
    name: exec
    description: resolving provider exec
    completedAt: 2021-01-03T21:55:49.786Z
    batchId: a6919a4c-49e3-4102-b3de-ef181790e47b
    output:
      name: exec
      dependencies: {}
      moduleConfigs: []
      config:
        name: exec
        dependencies: []
        path: /Users/samirpatel/htdocs/garden/examples/demo-project-start
      status:
        ready: true
        outputs: {}
        cached: true
      dashboardPages: []
    dependencyResults: {}
    version: 0.12.11
    startedAt: 2021-01-03T21:55:49.760Z
  resolve-provider.container:
    type: resolve-provider
    key: resolve-provider.container
    name: container
    description: resolving provider container
    completedAt: 2021-01-03T21:55:49.780Z
    batchId: 23e0dffd-ed63-4e04-8b9d-41988ec5c8e6
    output:
      name: container
      dependencies: {}
      moduleConfigs: []
      config:
        name: container
        dependencies: []
        path: /Users/samirpatel/htdocs/garden/examples/demo-project-start
      status:
        ready: true
        outputs: {}
        cached: true
      dashboardPages: []
    dependencyResults: {}
    version: 0.12.11
    startedAt: 2021-01-03T21:55:49.762Z
  resolve-provider.local-kubernetes:
    type: resolve-provider
    description: resolving provider local-kubernetes
    key: resolve-provider.local-kubernetes
    name: local-kubernetes
    startedAt: 2021-01-03T21:55:49.781Z
    completedAt: 2021-01-03T21:55:59.719Z
    batchId: 23e0dffd-ed63-4e04-8b9d-41988ec5c8e6
    version: 0.12.11
  resolve-provider.templated:
    type: resolve-provider
    key: resolve-provider.templated
    name: templated
    description: resolving provider templated
    completedAt: 2021-01-03T21:55:49.784Z
    batchId: e9d3f176-b4b2-46d3-99f4-bf8ff36b1a2d
    output:
      name: templated
      dependencies: {}
      moduleConfigs: []
      config:
        name: templated
        path: /Users/samirpatel/htdocs/garden/examples/demo-project-start
      status:
        ready: true
        outputs: {}
        cached: true
      dashboardPages: []
    dependencyResults: {}
    version: 0.12.11
    startedAt: 2021-01-03T21:55:49.763Z
messages: []


```

**Comment by rfuhrer** [comment]
Okay, glad to see it wasn't just me having this problem.  I was doing a bit of research and trying out Garden and couldn't get this to work.  I saw that helm deprecated that URL (https://kubernetes-charts.storage.googleapis.com) and moved it to https://charts.helm.sh/stable.  

I found this: https://stackoverflow.com/questions/65407317/helm-init-failed-is-not-a-valid-chart-repository-or-cannot-be-reached-failed-to

I was able to fix it by running the repo commands on the helm installed by garden:
```
C:\Users\roryf\.garden\tools\helm\b953dd95f12c100f\windows-amd64\helm.exe repo add stable https://charts.helm.sh/stable
C:\Users\roryf\.garden\tools\helm\b953dd95f12c100f\windows-amd64\helm.exe repo update
```
Obviously pull your path from your own error message
